docs/can-guides/topics/debugging.md

```diff
@@ -331,3 +331,99 @@ Component.extend({
     }
 });
 ```
+
+## Installing CanJS Devtools
+
+There is a CanJS Devtools Chrome Extension available in the [Chrome Web Store](https://chrome.google.com/webstore/detail/canjs-devtools/hhdfadlgplkpapjfehnjhcebebgmibcb).
+
+Once installed, you will see a CanJS icon next to the address bar:
+
+<img src="../../docs/can-guides/images/devtools/canjs-button-disabled.png"
+  alt="The CanJS Devtools extension icon - disabled"
+  width="600px"/>
```

Comment:
Did you try using `class="bit-docs-screenshot"` on these? Not sure if they’d make the images look better or not. 😃

Reply:
It centers them 😄.